spec/inquisition/tasks/inquisition_rake_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rake'
```

Comment:
Propose to create a `shared_context` with the next functionality:
- require rake task
- task -> returns the task
- define_task

Reply:
Propose to rename this file to `inquisition_spec.rb`